src/Components/test/testassets/BasicTestApp/InteropComponent.razor

```diff
@@ -175,6 +175,22 @@
             ReturnValues["roundTripByteArrayWrapperObjectFromDotNet"] = ((IJSInProcessRuntime)JSRuntime).Invoke<ByteArrayInterop.ByteArrayWrapper>("roundTripByteArrayWrapperObject", byteArrayWrapperObject).ToString();
         }
 
+        // Only run this test for Blazor Server
+        if (shouldSupportSyncInterop)
+        {
+            // Set the result to the expected value in the WASM case to ensure
+            // missing this test case doesn't result in test failure.
+            ReturnValues["jsToDotNetStreamAsync"] = "1,5,7,15,35,200";
```

Comment:
Is there a more canonical way to go about this instead of just setting the result to the expected result?

Reply:
Not sure what you mean by this comment, can you clarify?

Reply:
We probably want to use a larger payload than this. I would suggest you think of a way of generating a large payload here and in the JS side and use that. 

One way to go about it is to generate a byte array on the C# side and send it via JS interop ahead of time and then use the streaming support to read it back. (It should round-trip).

Reply:
I'm trying to have this E2E test run specifically for blazorserver (and NOT for WASM). Doing something like `!shouldSupportSyncInterop` isn't sufficient, as the test infra will still expect a `ReturnValues` for WASM. I got around this by just setting the result value in the WASM case, to the expected result value, thereby ignoring the test. The of course is far from ideal and I was wondering if there's an alternate / more common way to do this.

Reply:
Fair enough

Reply:
> We probably want to use a larger payload than this. 

Updated to a payload of a 100,000 bytes. Let me know if you'd suggest going even larger.